src/PeopleTable.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react';
+import { getPeople } from './api';
+import { Person } from './types/Person';
+import { Loader } from './components/Loader';
+
+export const PeopleTable = () => {
```

Comment:
Fetch people from [the API](https://mate-academy.github.io/react_people-table/api/people.json) when PeoplePage is opened;
Pass people to PeopleTable component and render as a table (see the given markup);